navigation/MainTabNavigator.js

```diff
@@ -6,19 +6,25 @@ import {
 } from 'react-navigation';
 
 import TabBarIcon from '../components/TabBarIcon';
-import HomeScreen from '../screens/HomeScreen';
+import ChatsScreen from '../screens/ChatsScreen';
+import ChatScreen from '../screens/ChatScreen';
 import SettingsScreen from '../screens/SettingsScreen';
 
-const HomeStack = createStackNavigator({
-    Home: HomeScreen
+const ChatsStack = createStackNavigator({
```

Comment:
we use STACK navigator here because Chats and Chat are related... you click on a chat from the chats screen to access it.